database/seeders/DatabaseSeeder.php

```diff
@@ -13,6 +14,7 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
+        GameFactory::factory(10)->create();
```

Comment:
i don't think we need this here

Reply:
removed and resolved in most recent push!